docs/examples/custom-secrets/README.md

```diff
@@ -0,0 +1,25 @@
+# Local Secrets Monitoring
+
+If you have local secrets you wish to monitor, the examples in this directory will help you acheive that.
```

Comment:
```suggestion
If you have local secrets you wish to monitor, the examples in this directory will help you achieve that.
```

Reply:
Good catch! Can’t believe I wrote that lol. I’ll update and send your way tomorrow. Thanks!